INSTALL.md

```diff
@@ -0,0 +1,84 @@
+# Installing device plugins to cluster
+
+## Install device plugins via a DaemonSet
+
+Each plugin can be installed via a DaemonSet. The install changes slightly based on the desired plugin. See install instructions per [plugin](README.md#plugins).
+
+Installing plugins via DaemonSets deployes them to the ```default``` (or currently active) namespace. Use kubectl's ```--namespace``` argument to change the deployment namespace.
+
+## Install device plugins via device plugin operator
+
+A more advanced install method is via device plugin operator. Operator configures plugin deployments based on the supplied device plugin CRDs (Custom Resource Definitions). See installation instructions in the [operator README](cmd/operator/README.md#installation).
+
+Operator installs device plugins to the ```inteldeviceplugins-system``` namespace.
```

Comment:
```suggestion
Operator installs device plugins to the same namespace where the operator itself is deployed.
```

Reply:
Sure. Though, I'd still add something like "Operator's default namespace is.." afterwards.